rllab/envs/gym_env.py

```diff
@@ -56,7 +56,7 @@ def __call__(self, count):
 
 
 class GymEnv(Env, Serializable):
-    def __init__(self, env_name, record_video=True, video_schedule=None, log_dir=None, record_log=True,
+    def __init__(self, env, record_video=True, video_schedule=None, log_dir=None, record_log=True,
```

Comment:
Does this work when you use run_experiment_lite() to run your algorithm (e.g. https://github.com/ryanjulian/rllab/blob/integration/examples/trpo_cartpole_pickled.py)?

I'm afraid they might have used strings to ensure that the GymEnv is Serializable